### ISSUE TITLE
inference time

### ISSUE THREAD

**Issue by Dantju** [HRNet/HigherHRNet-Human-Pose-Estimation#42]
@leoxiaobin have u test the time

**Comment by lzyyzlwsldx** [comment]
Have u test, I tried it needs about 2 sec/pic.

> @leoxiaobin have u test the time

**Comment by hellojialee** [comment]
@lzyyzlwsldx Hi, on multi-scale or single-scale in your test?

**Comment by stefanopini** [comment]
I have implemented a simplified version of HigherHRNet (compatible with official pretrained weights) providing a live demo script: [https://github.com/stefanopini/simple-HigherHRNet](https://github.com/stefanopini/simple-HigherHRNet)
My implementation runs at about 3 fps on a GTX 1060 (single scale).

**Comment by YangJae96** [comment]
stefanopini. Thank you very much!!

**Comment by hellojialee** [comment]
@stefanopini Great! Dose 3fps include flip test?

**Comment by stefanopini** [comment]
@hellojialee I'm sorry it doesn't, I still haven't implemented it.
However, predictions on original and flipped images are combined before most of the post-processing so the inference time should be faster than 1.5fps.

**Comment by hellojialee** [comment]
@stefanopini Yes, I agree. Thank you for answering!

**Comment by Dantju** [comment]
@leoxiaobin @stefanopini @hellojialee have u test image using HigherHRNet，it seems that it‘s demo results is  worser than centernet

**Comment by stefanopini** [comment]
@Dantju looking at the results reported in the papers, HigherHRNet should perform slightly better than CenterNet.
However, the top-down HRNet outperforms the HigherHRNet (see Table 2 in HigherHRNet paper) and in my experience it is quite faster.

CenterNet https://arxiv.org/pdf/1904.07850.pdf 
HigherHRNet https://openaccess.thecvf.com/content_CVPR_2020/papers/Cheng_HigherHRNet_Scale-Aware_Representation_Learning_for_Bottom-Up_Human_Pose_Estimation_CVPR_2020_paper.pdf

**Comment by Dantju** [comment]
@stefanopini yes in paper HIgherHRNet is better than centernet,but i use centernet and higherHRNet to test same image,it seems that the demo result of centernet is better than HigherHRNet，I confused

**Comment by seekFire** [comment]
@Dantju I think I found the code mistake that caused the bad result, see [here](https://github.com/HRNet/HigherHRNet-Human-Pose-Estimation/issues/17#issuecomment-785770361)
